test/run-tests

```diff
@@ -690,6 +695,16 @@ def handle_task(gh, args, config, task, dry_run=False):
         sys.exit(1)
 
 
+def sanity():
```

Comment:
Just noticed, it is better to name methods with a verb, for example `check_environment()`.